src/components/Alert/Alert.js

```diff
@@ -2,14 +2,36 @@
 Copyright (C) 2018 The Trustees of Indiana University
 SPDX-License-Identifier: BSD-3-Clause
 */
-import * as classNames from 'classnames';
-import * as React from 'react';
+import classNames from 'classnames';
+import React from 'react';
 
 import * as Rivet from '../util/Rivet';
 import Icon from '../util/RivetIcons';
-import { StatelessAlertProps, statelessPropTypes } from './common';
+import PropTypes from 'prop-types';
```

Comment:
I'm not saying all our imports will be perfect with so many changes and don't need to update here, but as a rule, we usually have external dependencies (in alpha order), followed by a blank line, followed by local dependencies (possibly followed by a blank line then CSS imports)

Reply:
👍 Move Prop-Types import to be higher